.github/workflows/rspec_tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           - {os: ubuntu-18.04, ruby: 2.5}
           - {os: ubuntu-18.04, ruby: 2.6}
           - {os: ubuntu-18.04, ruby: 2.7}
-          - {os: ubuntu-18.04, ruby: jruby-9.2.9.0}
+          - {os: ubuntu-18.04, ruby: jruby-9.2.17.0}
```

Comment:
@justinstoller I believe your PR bumped this to 9.2.10. Any reason not to go all the way to 9.2.17?

Reply:
I put it to 2.9.10.0 in my PR because that version should be the _oldest_ that works with the forge changes